Formula/metashell.rb

```diff
@@ -19,26 +19,25 @@ class Metashell < Formula
   end
 
   depends_on "cmake" => :build
+  depends_on "python@3.11" => :build
 
+  uses_from_macos "libedit"
+  uses_from_macos "libxml2"
   uses_from_macos "zlib"
 
   on_linux do
     depends_on "readline"
   end
 
   def install
-    ENV.cxx11
-
     # Build internal Clang
-    mkdir "3rd/templight/build" do
-      system "cmake", "../llvm", "-DLLVM_ENABLE_TERMINFO=OFF", *std_cmake_args
-      system "make", "templight"
-    end
+    system "cmake", "-S", "3rd/templight/llvm", "-B", "llvm_build", "-DLLVM_ENABLE_TERMINFO=OFF", *std_cmake_args
```

Comment:
This is changing the build directory of templight (the clang build shipped with metashell). Metashell's own build scripts are sensitive to the location of the templight builds, which is likely to be the reason for the error you get from the CI. Take a look at `CMakeLists.txt` in the root of Metashell's source tree and `cmake/Modules/FindTemplight.cmake` for more details.